### PR TITLE
Studentredux

### PR DESCRIPTION
**Pull request by jes708**

### CODE REVIEW

File: browser/js/components/StudentCard/index.js [hunk 1]
```diff
@@ -8,48 +8,97 @@ import { Card, CardActions, CardHeader, CardText } from 'material-ui/Card'
...
         </div>
       </Card>
     )
   }
 }
+
+const mapStateToProps = state => {
+  const { studentTestInfo } = state
+  const { isFetching } = studentTestInfo
+  console.log("AT LEAST YOU GOT THIS FAR", studentTestInfo.byId)
```
Comment (jdhang) [inline browser/js/components/StudentCard/index.js]: chop chop

File: browser/js/reducers/assessmentTeam.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ export default function assessmentTeam (state = initialState, action) {
         teamFetching: true
       })
     case LOAD_TEAM_SUCCESS:
+      console.log(action.team)
```
Comment (jdhang) [inline browser/js/reducers/assessmentTeam.js]: chop chop 🔪

File: browser/js/components/Comment/test.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ import {
   RadioButtonGroup
 } from 'material-ui/RadioButton'
 import Grade from '../../containers/Grade'
-const { cloneDeep as _cloneDeep } = require('lodash');
+import { cloneDeep as _cloneDeep } from 'lodash';
```
Comment (jancodes) [inline browser/js/components/Comment/test.js:22]: 👍 es6

File: server/env/development.js [hunk 4]
```diff
@@ -6,5 +6,5 @@ module.exports = {
     "clientSecret": "e79eb4cec3230600099c393a4f5abf0f899ca11b",
     "callbackURL": "http://127.0.0.1:1337/auth/github/callback"
   },
-  "LOGGING": false
+  "LOGGING": true
```
Comment (jancodes) [inline server/env/development.js:9]: let's keep logging true 👍

Comment (jdhang) [inline server/env/development.js:9]: 🔪 chop chop

**Pull request merged**